.github/workflows/ci.yml

```diff
@@ -15,7 +15,7 @@ jobs:
       fail-fast: false
       matrix:
         java: ['adopt@1.8', 'adopt@1.11']
-        scala: ['2.12.12', '2.13.4']
+        scala: ['2.12.12', '2.13.6']
```

Comment:
Might want to update to `2.12.14` while you're here. Also could be good to try to add `2.11` support.

Reply:
Added it to the build file - will see how the 2.11 goes

Reply:
I think there is only one test that is using JSON so could potentially pull that into a version specific file but also fine to do as a follow up.